### ISSUE TITLE
Control 2 chargers on same circuit

### ISSUE THREAD

**Issue by Mats1717** [fondberg/easee_hass#357]
### Question

I have 2 chargers on the same circuit so controlling them idividually by circuit_dynamic_limit is not possible.
What is the best way to cotroll them individually in a sheduling automation. I will not run them on the same time due to effect fee

### What version of the integration are you using?

Latest

**Comment by olalid** [comment]
I guess setting the charger dynamic limit would be one way. It is automatically reset though when charging starts so you would have to make sure to set it when charging starts or just once a minute or so (if you call the service it will only call the API if the limit is different from what is already set, so no harm in doing that).

But, also if you want to limit power, setting the circuit dynamic limit does mean that the power is limited for both chargers total power. I.e if you set the limit to anything less than 12A then only one charger will start since the minimum current for one charger to start is 6A. You would know which one of the charger starts fist though.

A third way would be to pause the charger you do not want to start...

I do not have this kind of setup myself so I have never tried any of these methods.

**Comment by Mats1717** [comment]
What do you think of using switch.easee_garaget_charger_enabled.
Is that a bad idea.

**Comment by olalid** [comment]
It should work I guess, but since it is a charger configuration setting it might be so that it is written to flash in the charger. So there is a risk of wearing out the flash memory if the setting is changed too often.

**Comment by AMajland** [comment]
I am seeing something a bit weird on a circuit with two chargers on the same circuit.  On the pictures the left is a master and the right is a slave.

The slave is limited to 10A in the static charger limit
Dyn charger,  Dyn circuit, Max Charger, Max Circuit, Offline 
![image](https://github.com/nordicopen/easee_hass/assets/97510471/3ef67cd2-1148-46bc-b963-5d3edebd35c8)

As expected the master is using 32A as the limit and the slave 10A

But when i set the Dyn Curcuit Limit to i.e 6A it is only reflected on the master and not on the slave.

![image](https://github.com/nordicopen/easee_hass/assets/97510471/bd2f4460-d73c-4bee-a482-0af04840d8ca)

Looking at the attributes of the slaves dynamic_circuit_limit they are newer changed by calling the easee.set_circuit_dynamic_limit with the circuit_id: as data. Only the masters dynamic_circuit_limit  are updated.

Both easee home chargers are at firmware v318

I was just helping another with two chargers and two cars that wanted to charge with surplus from solar and had issues. I have not testet with my own car yet whether the slaves honors the circuit limit of i.e 6A in this example when it shows that it will use the higher Max Charger Limit

**Comment by AMajland** [comment]
Yes they are on the same circuit - Both shown in easee.cloud and have the same circuit_id shown under 
 dynamic_circuit_limit  for both chargers.
 
 I'll test how it behaves with a car connected tonight when it gets home.

**Comment by Mats1717** [comment]
It will use the value of the master but i dont think it will update anyway when charging.
I use charger_enabled for automation by schedule so it wont update so often.
If you have 32 A circuit you can set upp both as master and the then it will work if you have cars that only can charge max 16 A AC

**Comment by olalid** [comment]
We know since before that the circuit_current sensor does not update on slave chargers (so we decided some time ago to remove it on slave chargers).
Perhaps we should do the same with dynamic_circuit_limit on slave chargers if does not actually work...
Are you aware of other sensors that does not work on slaves?

"calculated" is not a sensor name that is provided by this integration, unless you have renamed it, so not sure what you mean there...? As I stated above, the sensor called "output_limit" will show you what limit is actually used by the charger. Hopefully that works on both master and slave. But it will always be 0 until charging starts. It might provide some clue to what limit is actually used.

**Comment by enoch85** [comment]
Didn't we already discuss this? https://github.com/nordicopen/easee_hass/issues/256#issuecomment-1774147335

**Comment by AMajland** [comment]
The car got home and we are now after 21 where the transport tarifs goes from  1.2388 DKK/kWh to 0,4129 DKK (~0,17€ €/KWh to  ~0,06 €/KWh)

- The car is connected
- The dynamic circuit max is set at 0
- The slave shows dynamic circuit max on the attribute of CHARGERID_dynamic_circuit_limit as 40 (as above)

And the slave charger status is PAUSED/AWAITING START with the reason MAX DYNAMIC CIRCUIT TO LOW

Setting the dynamic circuit max to 6A starts charging at the slave with "CHARGERID_output_limit at 6A

So all works as expected but i was just puzzled by the slave showing nonsens values for dynamic_circuit_max.

The "calculated" is in the https://github.com/tmjo/charger-card and does not take into account that the dynamic max for a slave is not updated.

Master (nothing connected)
![image](https://github.com/nordicopen/easee_hass/assets/97510471/ec134fdd-22a5-4fbc-a8b6-47db07083e1d)

Slave:
![image](https://github.com/nordicopen/easee_hass/assets/97510471/c0a7f871-83e5-49da-8c5b-e99cca5c65bf)

**Comment by olalid** [comment]
See https://github.com/nordicopen/easee_hass/pull/384

**Comment by olalid** [comment]
The non-working sensors mentioned above are now removed in the latest release.
